test/api_calls.js

```diff
@@ -955,6 +955,9 @@ it('should let buyers set their first use time or use a default one if not set',
     assert.equal(ethBalanceBefore.minus(creditAmount).minus(weiConsumedByGas).eq(ethBalanceAfter), true)
     assert.equal(addCreditsTxn.logs[0].event, 'LogDepositCredits')
 
+    ethBalanceBefore = await web3.eth.getBalance(accounts[7])
```

Comment:
I did this fix because following asserts for `accoutns[7]` balance and credits were trying to compare `accounts[7]` after credit/balance with corresponding values of `accounts[6]` before credit/balance.